src/touchstone/databases/elasticsearch.py

```diff
@@ -71,8 +71,9 @@ def emit_compute_dict(self, uuid, compute_map, index, identifier):
 
         logger.debug("Initializing search object")
         kw_identifier = identifier + ".keyword"  # append .keyword
-        s = Search(using=self._conn_object, index=str(index)).query("match", **{kw_identifier: uuid})
-
+        #s = Search(using=self._conn_object, index=str(index)).query("match", **{kw_identifier: uuid})
+         s= elasticsearch.helpers.scan.(self._conn_object, query={"query": {"match", **{kw_identifier: uuid}}}, index=str(index), scroll='1m')    
```

Comment:
spacing seems off, additionally - does this break any previous queries?

Reply:
Forgot to comment the first 's' out. Making a few for changes to the code, will put it in.